src/lib/messenger/passback.ts

```diff
@@ -199,7 +199,7 @@ const init = (register: RegisterListener): void => {
 							if (slotId === passbackElementId) {
 								const size = event.size;
 								if (Array.isArray(size)) {
-									const adHeight = size[1];
+									const adHeight = Number(size[1]);
```

Comment:
I think there might be two options here which would avoid the case where you call `Number(undefined)` at runtime (which may or may not be possible, depending on the assumptions below):

1) Assume that googletag may return improperly sized arrays and perform the check in the if statement:

```typescript
if (Array.isArray(size) && size[1]) {
  // ...
}
```

2) Assume googletag always returns an array of the correct size, and either ignore the rule or use a non-null assertion:

```typescript
const adHeight = size[1]!;
```

Note this will generate an eslint warning, which could be ignored / disabled with a comment.

Reply:
Great shout, I'll implement option 1